pkg/tsdb/cloudwatch/get_dimension_values_for_wildcards_test.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 func TestGetDimensionValuesForWildcards(t *testing.T) {
-	logger := &logtest.Fake{}
+	testLogger := log.NewNopLogger()
```

Comment:
I renamed the variable for the tests in this package because we actually have a `logger` global for the real logger